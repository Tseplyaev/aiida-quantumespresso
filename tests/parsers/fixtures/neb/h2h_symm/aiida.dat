      0.0000000000      0.0000000000      2.1764284244
      0.2255889029     -0.1137301503      0.0126032441
      0.5000000043     -0.2847563767      0.0187003162
      0.7744111027     -0.1137301491      0.0126033840
      1.0000000000      0.0000000006      2.0996346522
