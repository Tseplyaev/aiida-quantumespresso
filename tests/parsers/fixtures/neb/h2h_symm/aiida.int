      0.0000000000      0.0000000000
      0.0040000000     -0.0000903488
      0.0080000000     -0.0003574533
      0.0120000000     -0.0007954008
      0.0160000000     -0.0013982784
      0.0200000000     -0.0021601735
      0.0240000000     -0.0030751731
      0.0280000000     -0.0041373647
      0.0320000000     -0.0053408353
      0.0360000000     -0.0066796723
      0.0400000000     -0.0081479628
      0.0440000000     -0.0097397941
      0.0480000000     -0.0114492534
      0.0520000000     -0.0132704279
      0.0560000000     -0.0151974048
      0.0600000000     -0.0172242715
      0.0640000000     -0.0193451150
      0.0680000000     -0.0215540227
      0.0720000000     -0.0238450818
      0.0760000000     -0.0262123795
      0.0800000000     -0.0286500029
      0.0840000000     -0.0311520395
      0.0880000000     -0.0337125763
      0.0920000000     -0.0363257006
      0.0960000000     -0.0389854997
      0.1000000000     -0.0416860607
      0.1040000000     -0.0444214709
      0.1080000000     -0.0471858175
      0.1120000000     -0.0499731877
      0.1160000000     -0.0527776689
      0.1200000000     -0.0555933481
      0.1240000000     -0.0584143126
      0.1280000000     -0.0612346498
      0.1320000000     -0.0640484466
      0.1360000000     -0.0668497905
      0.1400000000     -0.0696327687
      0.1440000000     -0.0723914683
      0.1480000000     -0.0751199766
      0.1520000000     -0.0778123808
      0.1560000000     -0.0804627681
      0.1600000000     -0.0830652259
      0.1640000000     -0.0856138412
      0.1680000000     -0.0881027014
      0.1720000000     -0.0905258936
      0.1760000000     -0.0928775051
      0.1800000000     -0.0951516232
      0.1840000000     -0.0973423349
      0.1880000000     -0.0994437277
      0.1920000000     -0.1014498886
      0.1960000000     -0.1033549050
      0.2000000000     -0.1051528640
      0.2040000000     -0.1068378528
      0.2080000000     -0.1084039588
      0.2120000000     -0.1098452692
      0.2160000000     -0.1111558711
      0.2200000000     -0.1123298517
      0.2240000000     -0.1133612984
      0.2280000000     -0.1143018090
      0.2320000000     -0.1153798543
      0.2360000000     -0.1166151806
      0.2400000000     -0.1180025773
      0.2440000000     -0.1195368336
      0.2480000000     -0.1212127389
      0.2520000000     -0.1230250825
      0.2560000000     -0.1249686538
      0.2600000000     -0.1270382419
      0.2640000000     -0.1292286363
      0.2680000000     -0.1315346262
      0.2720000000     -0.1339510010
      0.2760000000     -0.1364725500
      0.2800000000     -0.1390940625
      0.2840000000     -0.1418103277
      0.2880000000     -0.1446161351
      0.2920000000     -0.1475062740
      0.2960000000     -0.1504755336
      0.3000000000     -0.1535187033
      0.3040000000     -0.1566305723
      0.3080000000     -0.1598059301
      0.3120000000     -0.1630395658
      0.3160000000     -0.1663262689
      0.3200000000     -0.1696608286
      0.3240000000     -0.1730380343
      0.3280000000     -0.1764526753
      0.3320000000     -0.1798995408
      0.3360000000     -0.1833734203
      0.3400000000     -0.1868691029
      0.3440000000     -0.1903813781
      0.3480000000     -0.1939050351
      0.3520000000     -0.1974348633
      0.3560000000     -0.2009656520
      0.3600000000     -0.2044921904
      0.3640000000     -0.2080092680
      0.3680000000     -0.2115116740
      0.3720000000     -0.2149941977
      0.3760000000     -0.2184516284
      0.3800000000     -0.2218787555
      0.3840000000     -0.2252703683
      0.3880000000     -0.2286212561
      0.3920000000     -0.2319262081
      0.3960000000     -0.2351800138
      0.4000000000     -0.2383774625
      0.4040000000     -0.2415133433
      0.4080000000     -0.2445824458
      0.4120000000     -0.2475795591
      0.4160000000     -0.2504994726
      0.4200000000     -0.2533369756
      0.4240000000     -0.2560868574
      0.4280000000     -0.2587439074
      0.4320000000     -0.2613029148
      0.4360000000     -0.2637586690
      0.4400000000     -0.2661059593
      0.4440000000     -0.2683395749
      0.4480000000     -0.2704543053
      0.4520000000     -0.2724449397
      0.4560000000     -0.2743062674
      0.4600000000     -0.2760330778
      0.4640000000     -0.2776201601
      0.4680000000     -0.2790623038
      0.4720000000     -0.2803542980
      0.4760000000     -0.2814909321
      0.4800000000     -0.2824669955
      0.4840000000     -0.2832772774
      0.4880000000     -0.2839165671
      0.4920000000     -0.2843796541
      0.4960000000     -0.2846613275
      0.5000000000     -0.2847563767
      0.5040000000     -0.2846613279
      0.5080000000     -0.2843796548
      0.5120000000     -0.2839165683
      0.5160000000     -0.2832772789
      0.5200000000     -0.2824669973
      0.5240000000     -0.2814909343
      0.5280000000     -0.2803543004
      0.5320000000     -0.2790623065
      0.5360000000     -0.2776201631
      0.5400000000     -0.2760330810
      0.5440000000     -0.2743062708
      0.5480000000     -0.2724449433
      0.5520000000     -0.2704543091
      0.5560000000     -0.2683395789
      0.5600000000     -0.2661059634
      0.5640000000     -0.2637586733
      0.5680000000     -0.2613029192
      0.5720000000     -0.2587439119
      0.5760000000     -0.2560868621
      0.5800000000     -0.2533369803
      0.5840000000     -0.2504994774
      0.5880000000     -0.2475795639
      0.5920000000     -0.2445824507
      0.5960000000     -0.2415133483
      0.6000000000     -0.2383774674
      0.6040000000     -0.2351800188
      0.6080000000     -0.2319262131
      0.6120000000     -0.2286212610
      0.6160000000     -0.2252703732
      0.6200000000     -0.2218787604
      0.6240000000     -0.2184516333
      0.6280000000     -0.2149942025
      0.6320000000     -0.2115116787
      0.6360000000     -0.2080092727
      0.6400000000     -0.2044921950
      0.6440000000     -0.2009656565
      0.6480000000     -0.1974348678
      0.6520000000     -0.1939050395
      0.6560000000     -0.1903813824
      0.6600000000     -0.1868691071
      0.6640000000     -0.1833734243
      0.6680000000     -0.1798995447
      0.6720000000     -0.1764526791
      0.6760000000     -0.1730380380
      0.6800000000     -0.1696608321
      0.6840000000     -0.1663262723
      0.6880000000     -0.1630395690
      0.6920000000     -0.1598059331
      0.6960000000     -0.1566305752
      0.7000000000     -0.1535187060
      0.7040000000     -0.1504755362
      0.7080000000     -0.1475062765
      0.7120000000     -0.1446161375
      0.7160000000     -0.1418103299
      0.7200000000     -0.1390940645
      0.7240000000     -0.1364725518
      0.7280000000     -0.1339510027
      0.7320000000     -0.1315346278
      0.7360000000     -0.1292286377
      0.7400000000     -0.1270382431
      0.7440000000     -0.1249686549
      0.7480000000     -0.1230250835
      0.7520000000     -0.1212127397
      0.7560000000     -0.1195368343
      0.7600000000     -0.1180025778
      0.7640000000     -0.1166151810
      0.7680000000     -0.1153798545
      0.7720000000     -0.1143018091
      0.7760000000     -0.1133612986
      0.7800000000     -0.1123298522
      0.7840000000     -0.1111558718
      0.7880000000     -0.1098452702
      0.7920000000     -0.1084039601
      0.7960000000     -0.1068378543
      0.8000000000     -0.1051528656
      0.8040000000     -0.1033549068
      0.8080000000     -0.1014498906
      0.8120000000     -0.0994437298
      0.8160000000     -0.0973423372
      0.8200000000     -0.0951516255
      0.8240000000     -0.0928775075
      0.8280000000     -0.0905258961
      0.8320000000     -0.0881027039
      0.8360000000     -0.0856138438
      0.8400000000     -0.0830652285
      0.8440000000     -0.0804627707
      0.8480000000     -0.0778123834
      0.8520000000     -0.0751199791
      0.8560000000     -0.0723914708
      0.8600000000     -0.0696327712
      0.8640000000     -0.0668497930
      0.8680000000     -0.0640484490
      0.8720000000     -0.0612346521
      0.8760000000     -0.0584143149
      0.8800000000     -0.0555933503
      0.8840000000     -0.0527776709
      0.8880000000     -0.0499731897
      0.8920000000     -0.0471858194
      0.8960000000     -0.0444214726
      0.9000000000     -0.0416860623
      0.9040000000     -0.0389855012
      0.9080000000     -0.0363257020
      0.9120000000     -0.0337125776
      0.9160000000     -0.0311520407
      0.9200000000     -0.0286500040
      0.9240000000     -0.0262123804
      0.9280000000     -0.0238450826
      0.9320000000     -0.0215540234
      0.9360000000     -0.0193451156
      0.9400000000     -0.0172242719
      0.9440000000     -0.0151974051
      0.9480000000     -0.0132704281
      0.9520000000     -0.0114492534
      0.9560000000     -0.0097397941
      0.9600000000     -0.0081479627
      0.9640000000     -0.0066796721
      0.9680000000     -0.0053408351
      0.9720000000     -0.0041373643
      0.9760000000     -0.0030751727
      0.9800000000     -0.0021601730
      0.9840000000     -0.0013982779
      0.9880000000     -0.0007954002
      0.9920000000     -0.0003574527
      0.9960000000     -0.0000903482
      1.0000000000      0.0000000006
