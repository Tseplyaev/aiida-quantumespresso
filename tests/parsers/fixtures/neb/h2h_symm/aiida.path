RESTART INFORMATION
      14
      20
       0
NUMBER OF IMAGES
   5
ENERGIES, POSITIONS AND GRADIENTS
Image:    1
       -1.6399117415
     -4.566700090012      0.000000000000      0.000000000000     -0.000384669442     -0.000000000000     -0.000000000000  1  0  0
      0.000000000000      0.000000000000      0.000000000000     -0.000000000000     -0.000000000000     -0.000000000000  0  0  0
      1.557766760017      0.000000000000      0.000000000000      0.042324797747     -0.000000000000     -0.000000000000  1  0  0
Image:    2
       -1.6440912478
     -3.292813624666      0.000000000000      0.000000000000     -0.001453144019     -0.000000000000     -0.000000000000
      0.000000000000      0.000000000000      0.000000000000      0.000000000000     -0.000000000000     -0.000000000000
      1.430609362120      0.000000000000      0.000000000000      0.000091426118     -0.000000000000     -0.000000000000
Image:    3
       -1.6503763463
     -1.773822002428      0.000000000000      0.000000000000      0.000363662102     -0.000000000000     -0.000000000000
      0.000000000000      0.000000000000      0.000000000000     -0.000000000000     -0.000000000000     -0.000000000000
      1.773822043374      0.000000000000      0.000000000000     -0.000363662102     -0.000000000000     -0.000000000000
Image:    4
       -1.6440912478
     -1.430609359139      0.000000000000      0.000000000000     -0.000091425625     -0.000000000000     -0.000000000000
      0.000000000000      0.000000000000      0.000000000000     -0.000000000000     -0.000000000000     -0.000000000000
      3.292813657335      0.000000000000      0.000000000000      0.001453143887     -0.000000000000     -0.000000000000
Image:    5
       -1.6399117415
     -1.557766760017      0.000000000000      0.000000000000     -0.042324705771     -0.000000000000     -0.000000000000
      0.000000000000      0.000000000000      0.000000000000      0.000000000000     -0.000000000000     -0.000000000000
      4.566700090012      0.000000000000      0.000000000000      0.000384644343     -0.000000000000     -0.000000000000
