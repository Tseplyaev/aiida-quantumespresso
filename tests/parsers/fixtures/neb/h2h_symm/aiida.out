
     Program NEB v.6.4.1 starts on 30Jul2019 at 16:28:31 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes

     No input file found, assuming nothing to parse
     Searching argument -input_images or --input_images
     Reading input from pw_1.in
     Reading input from pw_2.in

     initial path length           =  4.2553 bohr
     initial inter-image distance  =  1.0638 bohr

     string_method                 =    neb
     restart_mode                  =    from_scratch
     opt_scheme                    =    broyden
     num_of_images                 =    5
     nstep_path                    =    20
     CI_scheme                     =    auto
     first_last_opt                =    F
     use_freezing                  =    F
     ds                            =    2.0000 a.u.
     k_max                         =    0.3000 a.u.
     k_min                         =    0.2000 a.u.
     suggested k_max               =    0.1542 a.u.
     suggested k_min               =    0.1028 a.u.
     path_thr                      =    0.1000 eV / A

     ------------------------------ iteration   1 ------------------------------

     tcpu =      0.2    self-consistency for image   1
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      0.9    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      1.7    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      2.6    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      3.4    self-consistency for image   5
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   2.487921 eV
     activation energy (<-) =   2.487921 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -43.1027484            3.456929            F
         3        -42.1363465            2.305559            F
         4        -43.1027484            3.456926            F
         5        -44.6242679            2.176424            T

     climbing image =  3

     path length          =  4.255 bohr
     inter-image distance =  1.064 bohr

     ------------------------------ iteration   2 ------------------------------

     tcpu =      4.1    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      4.9    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      5.8    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   2.035026 eV
     activation energy (<-) =   2.035026 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -43.4535979            2.521306            F
         3        -42.5892422            2.460480            F
         4        -43.4535974            2.521307            F
         5        -44.6242679            2.176424            T

     climbing image =  3

     path length          =  4.319 bohr
     inter-image distance =  1.080 bohr

     ------------------------------ iteration   3 ------------------------------

     tcpu =      6.7    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      7.6    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =      8.4    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   1.372648 eV
     activation energy (<-) =   1.372648 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.0746978            2.554052            F
         3        -43.2516202            2.560761            F
         4        -44.0746976            2.554053            F
         5        -44.6242679            2.176424            T

     climbing image =  3

     path length          =  4.565 bohr
     inter-image distance =  1.141 bohr

     ------------------------------ iteration   4 ------------------------------

     tcpu =      9.4    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     10.2    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     10.9    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.682361 eV
     activation energy (<-) =   0.682361 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.5702779            2.192702            F
         3        -43.9419064            2.406379            F
         4        -44.5702779            2.192702            F
         5        -44.6242679            2.176424            T

     climbing image =  3

     path length          =  4.956 bohr
     inter-image distance =  1.239 bohr

     ------------------------------ iteration   5 ------------------------------

     tcpu =     11.8    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     12.5    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     13.2    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.064775 eV
     activation energy (<-) =   0.064775 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7494702            3.233209            F
         3        -44.5594927            1.780623            F
         4        -44.7494702            3.233210            F
         5        -44.6242679            2.176424            T

     climbing image =  3

     path length          =  5.454 bohr
     inter-image distance =  1.363 bohr

     ------------------------------ iteration   6 ------------------------------

     tcpu =     14.0    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     14.8    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     15.6    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.291238 eV
     activation energy (<-) =   0.291238 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.6895825            1.392725            F
         3        -44.3330296            2.097455            F
         4        -44.6895825            1.392725            F
         5        -44.6242679            2.176424            T

     climbing image =  3

     path length          =  5.120 bohr
     inter-image distance =  1.280 bohr

     ------------------------------ iteration   7 ------------------------------

     tcpu =     16.3    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     17.0    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     17.8    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7416807            1.955379            F
         3        -44.7298839            1.377760            F
         4        -44.7416806            1.955378            F
         5        -44.6242679            2.107932            T

     climbing image =  5

     path length          =  5.392 bohr
     inter-image distance =  1.348 bohr

     ------------------------------ iteration   8 ------------------------------

     tcpu =     18.5    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     19.2    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     19.9    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7257967            3.196330            F
         3        -44.8908259            0.548076            F
         4        -44.7257967            3.196331            F
         5        -44.6242679            2.043912            T

     climbing image =  5

     path length          =  5.795 bohr
     inter-image distance =  1.449 bohr

     ------------------------------ iteration   9 ------------------------------

     tcpu =     20.6    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     21.2    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     22.0    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7370164            0.242970            F
         3        -44.8748226            0.665490            F
         4        -44.7370164            0.242970            F
         5        -44.6242679            2.109288            T

     climbing image =  5

     path length          =  5.531 bohr
     inter-image distance =  1.383 bohr

     ------------------------------ iteration  10 ------------------------------

     tcpu =     22.6    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     23.1    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     23.8    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7364944            0.170083            F
         3        -44.8989943            0.372991            F
         4        -44.7364944            0.170082            F
         5        -44.6242679            2.109268            T

     climbing image =  5

     path length          =  5.591 bohr
     inter-image distance =  1.398 bohr

     ------------------------------ iteration  11 ------------------------------

     tcpu =     24.3    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     24.9    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     25.6    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7340052            0.493693            F
         3        -44.8914166            0.538525            F
         4        -44.7340052            0.493693            F
         5        -44.6242679            2.062655            T

     climbing image =  5

     path length          =  5.788 bohr
     inter-image distance =  1.447 bohr

     ------------------------------ iteration  12 ------------------------------

     tcpu =     26.2    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     26.7    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     27.4    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7376484            0.241910            F
         3        -44.9061263            0.211689            F
         4        -44.7376484            0.241910            F
         5        -44.6242679            2.096743            T

     climbing image =  5

     path length          =  5.713 bohr
     inter-image distance =  1.428 bohr

     ------------------------------ iteration  13 ------------------------------

     tcpu =     27.9    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     28.5    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     29.2    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7388231            0.843532            F
         3        -44.8648797            0.747780            F
         4        -44.7388231            0.843533            F
         5        -44.6242679            2.108890            T

     climbing image =  5

     path length          =  5.517 bohr
     inter-image distance =  1.379 bohr

     ------------------------------ iteration  14 ------------------------------

     tcpu =     29.8    self-consistency for image   2
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     30.3    self-consistency for image   3
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead
     tcpu =     31.0    self-consistency for image   4
     Message from routine setup:
     DEPRECATED: symmetry with ibrav=0, use correct ibrav instead

     activation energy (->) =   0.000000 eV
     activation energy (<-) =   0.000000 eV

     image        energy (eV)        error (eV/A)        frozen

         1        -44.6242679            2.176428            T
         2        -44.7379980            0.012603            F
         3        -44.9090243            0.018700            F
         4        -44.7379980            0.012603            F
         5        -44.6242679            2.099635            T

     climbing image =  5

     path length          =  5.675 bohr
     inter-image distance =  1.419 bohr

     ---------------------------------------------------------------------------


     neb: convergence achieved in  14 iterations

     NEB          :     29.36s CPU     31.59s WALL


   This run was terminated on:  16:29: 3  30Jul2019            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
